src/AppBundle/Resources/assets/scss/digideps/_alert.scss

```diff
@@ -1,5 +1,46 @@
 // =======================================================
 // ALERT
+.app-alert {
+    $alert-color: transparent;
+    @include govuk-responsive-padding(4);
+    @include govuk-responsive-margin(8, "bottom");
+    @include govuk-focusable;
+    max-width: 30em;
+    border-left: 5px solid transparent;
+}
+
+.app-alert__icon {
```

Comment:
You could make this section less verbose using &__icon and so on. (https://css-tricks.com/the-sass-ampersand/)

Reply:
Too much time spent writing vanilla CSS 😂 We're already using variables and inclusions so might as well commit.